sdk/tables/azure-data-tables/tests/test_table_async.py

```diff
@@ -242,7 +240,7 @@ async def test_list_tables_with_num_results(self, resource_group, location, stor
         async for s in ts.list_tables(results_per_page=3).by_page():
             small_page.append(s)
 
-        self.assertEqual(len(small_page), 2)
```

Comment:
why did this value change

Reply:
This test is fickle and i'm still working through it. Essentially here there are two pages and each page is appended to the list. I do not know how this one slipped through either, probably because they were running from playback instead of live.